projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1227,12 +1237,65 @@ function wpcom_launchpad_has_translation( $string, $domain = 'jetpack-mu-wpcom'
 	return $translated_string !== $string;
 }
 
+/**
+ * Determine `add_new_page` task visibility. The task is visible if there is no 'About' page on the site.
+ *
+ * @return bool True if we should show the task, false otherwise.
+ */
+function wpcom_launchpad_is_add_about_page_visible() {
+	return ! wpcom_is_update_about_page_task_visible();
```

Comment:
Given that the `_wp_layout_category` meta field is getting added in the Editor Toolkit plugin, we should probably add a check here for the existence/registration of that meta field.
```suggestion
	return ! wpcom_is_update_about_page_task_visible() && registered_meta_key_exists( 'post', '_wp_layout_category', 'page' );
```